docs/AboutUs.md

```diff
@@ -2,58 +2,59 @@
 layout: page
 title: About Us
 ---
+# About InternBuddy
```

Comment:
Currently, there seems to be inconsistency between whether we are going with `InternBuddy` or `Intern Buddy`. Let's standardise before accepting the pull request.

Reply:
We have decided to proceed with `InternBuddy` for naming conventions.